src/index_runner/main.py

```diff
@@ -35,13 +37,25 @@ def main():
     # All worker groups to send kafka messages to
     receivers = [es_indexers, releng_importers]
 
+    # used to check update every minute
+    last_updated_minute = int(time.time()/60)
+    _CONFIG_TAG = _query_for_config_tag("placeholder_tag")
```

Comment:
I think it'd be a little more clear if you removed the `existing_config_tag` param and had `_query_for_config_tag` return None if it fails to get a tag. In your condition below to check the tag, it can be `if config_tag and config_tag != _CONFIG_TAG`.